src/Knp/Menu/MenuItem.php

```diff
@@ -52,14 +52,14 @@ class MenuItem implements ItemInterface
     /**
      * Attributes for the item
      *
-     * @var array
+     * @var array<string, mixed>
```

Comment:
for these 4 `*attributes` properties, values can be specified more precisely than `mixed`. they are `string|bool|null`